### PR TITLE
Setup and mobile first

### PR DESCRIPTION
**Pull request by sanieni6**
## First version of my portfolio project
These are the initial features:
+ Add the index.html file
+ Add the styles.css file
+ Add the folders with images resources
+ Add a Readme.md file and license

### CODE REVIEW

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @sanieni6 👋, 

198009982-d99c2507-f20d-4c00-90a3-a54097598b36


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!🤗

## Highlights✅
- [x]  Commit messages are descriptive and meaningful👍.
- [x]  Pull request has a descriptive title and summary👍.
- [ ] Linters are properly setup👍. 

## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
## Milestone 1
Changes done:
+ Add the link to the homepage
+ Solve the overflowing problem

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <img src="images/menu.png" alt="menu button">
+
+    </nav>
+  
+  </header>
+  <section class="headline">
+    <h1 class="introduction">Hey There.
+      I'm Luis
+    </h1>
+    <h1 class="supporting-text">I'm a software developer</h1>
```
Comment (amejid) [inline index.html:23]: - [ ] [OPTIONAL] It is considered a best practice to have a single `h1` element per page. I highly recommend you replace the second `h1` element with an appropriate `hx` tag. (i.e. `h2`, `h3`,...)
Kindly check [this article](https://www.reliablesoft.net/h1-tag/) to get an insight into how to utilize `h1` element effectively. 🤝️

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Luis Sanchez</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html"><h1 class="name">Luis Sanchez</h1></a>
```
Comment (amejid) [inline index.html:12]: - [ ] [OPTIONAL] Kindly replace this `h1` element with a `p` or remove it completely as we are trying to maintain a single `h1` element per page. 🥸️

**Example**
```
<a href="index.html">Luis Sanchez</a>
```

**Review by amejid — APPROVED** [review]
Hi @sanieni6, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct use of Github flow :heavy_check_mark: 
- Linter checks passing :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**